instrumentation/src/javatest/org/oppia/android/instrumentation/EndToEndTestHelper.kt

```diff
@@ -0,0 +1,37 @@
+package org.oppia.android.instrumentation
+
+import android.content.Context
+import android.content.Intent
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.Until
+import org.junit.Assert
+
+/** This object contains common operations used for end-to-end tests. */
+object EndToEndTestHelper {
+
+  private val OPPIA_PACKAGE = "org.oppia.android"
+  private val LAUNCH_TIMEOUT = 30000L
+
+  /** Starts oppia from HomeScreen. */
+  fun UiDevice.startOppiaFromScratch() {
+    // Start from the home screen
+    this.pressHome()
```

Comment:
Nit: 'this' not needed (it's implied). Ditto elsewhere.

Reply:
Removed.

Reply:
It still seems present in the latest code changes, or am I missing something?

Reply:
Seems resolved now.